ws/DiscordWebsocket.ts

```diff
@@ -81,7 +81,8 @@ export default class DiscordWebsocket {
 
   restart(newSession = false) {
     if (newSession) this.sessionID = null;
-    this.disconnect().connect();
+    this.ws?.off('close', this._onClose);
```

Comment:
Would be better if this was placed in disconnect

Reply:
Makes sense